pkg/kubelet/metrics/metrics.go

```diff
@@ -241,7 +241,7 @@ var (
 			Help:           "Cumulative number of pod evictions by eviction signal",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"eviction_signal"},
+		[]string{"eviction_signal", "namespace"},
```

Comment:
this dramatically increases the cardinality of this metric and does not seem safe

Reply:
Ah, agreed, apparently we've been down this road before: https://github.com/kubernetes/kubernetes/pull/104033